### ISSUE TITLE
[COGNITION] 508-defect-3: Telephone numbers SHOULD be well-formed and use the accessible React component

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15046]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

* As a screen reader user, I want to hear telephone numbers read aloud in a steady cadence so I can understand them better.
* As a user who has a hard time "chunking" phone numbers when there are no dashes or separations, I want to see phone numbers with well-formatted blocks so I can recognize them as phone numbers easier.

## Details

Some of our telephone numbers (Express Care) are coming through as long strings of text without dash separators. Our content guidelines indicate we should be [separating phone numbers with dashes](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers) for easier pattern recognition.

> Use hyphens between numbers, and don’t use parentheses to set off the area code: 212-123-1234.

I'd also like to consider using the [Telephone design system component](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/telephone/) because it offers a better experience for screen readers by reading each number aloud individually.

## Acceptance Criteria

- [ ] Phone numbers are formatted correctly
- [ ] Team has implemented the Telephone design system component for appointment cards **OR**
- [ ] Team decides against using the Telephone component and documents the decision

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/past
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/express-care
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-express-care-request
* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-express-care-request/confirmation

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-10-20 at 9 14 55 AM](https://user-images.githubusercontent.com/934879/96601733-ef6ac880-12b7-11eb-83c4-aa89408ba7fb.png)

---
![Screen Shot 2020-10-20 at 9 15 02 AM](https://user-images.githubusercontent.com/934879/96601738-f0035f00-12b7-11eb-86d7-ac9e5196f1bf.png)

---
![Screen Shot 2020-10-20 at 9 15 44 AM](https://user-images.githubusercontent.com/934879/96601743-f09bf580-12b7-11eb-831f-f7d22a094cc8.png)

---
![Screen Shot 2020-10-20 at 9 14 49 AM](https://user-images.githubusercontent.com/934879/96601744-f1cd2280-12b7-11eb-8ddb-c091e504b7bf.png)

---
![Screen Shot 2020-10-20 at 9 35 02 AM](https://user-images.githubusercontent.com/934879/96601748-f1cd2280-12b7-11eb-972e-48cfc36dfd00.png)

**Comment by LanceGundersen** [comment]
@1Copenut the changes are on staging for your review.

**Comment by 1Copenut** [comment]
💯 This looks good in Staging. I tested with VoiceOver for a spot check and confirmed no axe violations with Chrome. Moving to close.
